env0/resource_cloud_credentials_project_assignment_test.go

```diff
@@ -90,20 +97,41 @@ func TestUnitResourceCloudCredentialsProjectAssignmentResource(t *testing.T) {
 			mock.EXPECT().RemoveCloudCredentialsFromProject(assignment.ProjectId, assignment.CredentialId).Times(1).Return(nil)
 		})
 	})
-	t.Run("Read didnt api didnt return correct stuff", func(t *testing.T) {
+	t.Run("detect drift", func(t *testing.T) {
```

Comment:
This tests that when assignment changes then we update its values.
What about a test that asserts we removes the item from the state (i.e `d.SetId("")`?

Reply:
all the tests for drift scenario are the same, I create a resource with specific values and then call to update the same resource with other values, but when reading the state before `update`, I return values that cause destroy.
the tests show that no update was called and another create is invoked, same as the real behavior of TF when drift detected